projects/storefrontlib/src/cms-components/navigation/navigation/navigation-ui.component.ts

```diff
@@ -91,6 +91,21 @@ export class NavigationUIComponent implements OnDestroy {
     event.stopPropagation();
   }
 
+  open(event: UIEvent): void {
```

Comment:
maybe instead of creating a whole new function, we could combine it with `toggleOpen()`